app/assets/stylesheets/views/article.scss

```diff
@@ -0,0 +1,364 @@
+@import '../config/import';
+
+:root {
+  --article-rythm: #{$su-2};
+
+  @media (min-width: $breakpoint-m) {
+    --article-rythm: #{$su-4};
+  }
+}
+
+.crayons-article {
+  &__cover {
+    margin: -#{$su-6} -#{$su-8} #{$su-7} -#{$su-8};
+    width: calc(100% + #{$su-8} + #{$su-8});
+    padding-top: percentage(
+      $cover-image-height / $cover-image-width
+    ); // TODO: fix ratio
+    background-size: cover;
+    background-position: center center;
+    border-radius: $br-default $br-default 0 0;
+  }
+
+  &__header {
+    margin-bottom: $su-6;
+  }
+
+  &__tags {
+    // TODO: replace with actual component one day...
+    font-family: $ff-accent;
+    font-size: $fs-s;
+    display: flex;
+  }
+
+  &__body {
+    font-size: $fs-l;
+    font-family: $ff-serif;
+    @media (min-width: $breakpoint-m) {
+      font-size: $fs-xl;
+    }
+
+    a {
+      code {
+        color: var(--link-color);
+      }
+
+      &.anchor {
+        padding-top: 0;
+        margin-top: 0;
+        top: -50px;
+        display: block;
+        visibility: hidden;
+        position: relative;
+      }
+    }
+
+    h1,
```

Comment:
Normally I thought that headings, tables, p styles etc. all  get defined at a high level app wide and we only override  a few if we want the style to change for that component. Do we approach it differently with our styling?

Reply:
That's usually true for blogs and other content heavy places that don't really contain other types of information and data...

DEV, even though is content heavy (articles) it also has pretty complex structure and data schema so we often use headings, paragraphs etc in many various ways - not only as representation of text-based content within post.

I think it's more common to actually "reset" styling for all kinds of elements globally and then redefine the styles for certain areas, like in article view. So we can keep using these elements in other places without inheriting original styling.